sponsor/models.py

```diff
@@ -24,3 +24,16 @@ def logo_display_for_admin(self):
             return "No logo"
 
     logo_display_for_admin.allow_tags = True
+
+
+class Donor(models.Model):
```

Comment:
We added a new model, but the PR does not contain the migration for the new model. We do that with `./manage.py makemigrations` :).